### PR TITLE
Removes a bunch of unnecessary lines

### PR DESCRIPTION
**Pull request by ChangelingRain**
advanced bullying. just look at that linecount.

### CODE REVIEW

File: code/datums/shuttles.dm [hunk 1]
```diff
@@ -147,50 +130,34 @@
...
 /datum/map_template/shuttle/emergency/imfedupwiththisworld
 	suffix = "imfedupwiththisworld"
 	name = "Oh, Hi Daniel"
-	description = "How was space work today? \
-		Oh, pretty good. We got a new space station and the company will make a lot of money. \
-		What space station? \
-		I cannot tell you; it's space confidential. \
-		Aw, come space on. Why not? \
-		No, I can't. Anyway, how is your space roleplay life?"
+	description = "How was space work today? Oh, pretty good. We got a new space station and the company will make a lot of money. What space station? I cannot tell you; it's space confidential. \
```
Comment (Iamgoofball) [inline code/datums/shuttles.dm]: this was intentionally separated as it's a conversation, I added this but not to comply with the meme text coiax did

Comment (ChangelingRain) [inline code/datums/shuttles.dm]: see above hyperfractal gigashuttle description

Comment (Iamgoofball) [inline code/datums/shuttles.dm]: yeah, I saw you fixed that one, but you kept the different people talking on different lines, this is a conversation between two people

i'll just fix it properly with newlines and shit for the shuttle description after this gets merged so whatever

Comment (ChangelingRain) [inline code/datums/shuttles.dm]: that's because that one used both actual quotes and newlines this is literally all on one line

Comment (Iamgoofball) [inline code/datums/shuttles.dm]: i'll fix it after this is merged

**Pull request merged**